transmogrifier/helpers.py

```diff
@@ -12,11 +12,26 @@
 from lxml import etree  # nosec B410
 from smart_open import open
 
+from transmogrifier.config import load_external_config
 from transmogrifier.models import TimdexRecord
 
 logger = logging.getLogger(__name__)
 
 
+def crosswalk_value(crosswalk_path: str, type_value: Optional[str]) -> Optional[str]:
```

Comment:
I’m adding this parameter and more specific crosswalk files given the likelihood of overlapping letter codes for other fields but let me know if you see a cleaner approach